format/format.go

```diff
@@ -360,6 +361,17 @@ func (f *fumpter) applyPre(c *astutil.Cursor) {
 	groupLoop:
 		for _, group := range node.Comments {
 			for _, comment := range group.List {
+				if trimmedText := strings.TrimSpace(comment.Text); trimmedText == "//gofumpt:diagnose" {
```

Comment:
I'd say just look for the prefix `//gofumpt:diagnose `; that way, we'll also update existing outdated comments, which should prevent confusion.

Reply:
ah yeah, thanks

but should we require the trailing space?